### PR TITLE
just: Update to v1.37.0

### PR DESCRIPTION
**Pull request by davidjharder**
**Summary**

- Release notes available [here](https://github.com/casey/just/releases/tag/1.37.0)

**Test Plan**

- Run some test commands from a test `justfile`

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/j/just/monitoring.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+releases:
+  id: 141393
+  rss: https://github.com/casey/just/tags
```
Comment (ermo) [inline packages/j/just/monitoring.yml:3]: should probably be `tags.atom`...?

**Review by ermo — CHANGES_REQUESTED** [review]
Apart from a small nit, this looks fine.

File: packages/j/just/monitoring.yml (hunk 1 above)
Comment (davidjharder) [reply packages/j/just/monitoring.yml:3]: yes

**Review by ermo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**